VndbSharp/Converters/AliasesConverter.cs

```diff
@@ -10,13 +11,23 @@ public override void WriteJson(JsonWriter writer, Object value, JsonSerializer s
 			throw new NotImplementedException();
 		}
 
-		public override Object ReadJson(JsonReader reader, Type objectType, Object existingValue, JsonSerializer serializer)
-		{
-			var values = reader.Value?.ToString().Split('\n');
```

Comment:
https://vndb.org/d11#5.3
Aliases on the "get producer" use comma separated values, so I figured it should be on the same converter because it is just a single character that is changed.